data/test/news/articles/c0000000025o.json

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
This article can be be called `c5jje4ejkqvo.json`, it doesnt need to be `c0000000025o.json`, e.g. `cl55zn0w0l4o.json` in this folder